backend/src/main/java/gov/cdc/usds/simplereport/config/AuthorizationProperties.java

```diff
@@ -22,4 +27,12 @@ public String getRoleClaim() {
   public String getRolePrefix() {
     return rolePrefix;
   }
+
+  public String getAdminGroupId() {
+    return adminGroupId;
+  }
+
+  public String getAdminGroupName() {
+    return adminGroupName;
```

Comment:
We should be able to derive this—I think the simplest route is to add a new property that just contains `SR-PROD-` and then have the admin group name and the tenant group prefix be derived from that (doesn't have to change the getters, just changes the constructor and adds a constant or two).